### PR TITLE
Able to pass authRequired on admin tests now, added preliminary tests

### PR DESCRIPTION
**Pull request by RyderBlues**
# Description
Fixes # (issue)

## Type of change
Please delete options that are not relevant.
- [ x ] New feature (non-breaking change which adds functionality)

## Change Status
- [ x ] Complete, tested, ready to review and merge

# Checklist
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: __tests__/routes/adminIncidents.test.js [hunk 1]
```diff
@@ -1,17 +1,31 @@
 const request = require('supertest');
-const db = require('../../data/db-config');
-const Incidents = require('../../api/allIncidents/incidentsModel');
-const server = require('../../api/allIncidents/adminIncidentsRouter');
+const server = require('../../api/app');
+const authRequired = require('../../api/middleware/authRequired');
+jest.mock('../../api/middleware/authRequired');
+
+// Allows you to pass Admin authentication
+beforeEach(() => {
```
Comment (alvin-joseph) [inline __tests__/routes/adminIncidents.test.js:7]: Nice work figuring out how to mock the authorization to run the tests! This is a nice solution!

**Review by alvin-joseph — APPROVED** [review]
All the tests and changes look great! Nice work!

File: api/allIncidents/incidentsModel.js [hunk 2]
```diff
@@ -19,7 +19,6 @@ module.exports = {
  */
 async function getIncidents() {
   const incidents = await db('force_ranks')
-    .where({ status: 'approved' })
```
Comment (joseph-witzke) [inline api/allIncidents/incidentsModel.js:22]: Did you remove this because our local DB has incidents with statuses other than approved?

**Review by joseph-witzke — APPROVED** [review]
Nice! So it looks like the key was adding the auth middleware to the jest.mock and then running it before each?

**Pull request merged**